### ISSUE TITLE
Clicking on mayor in venue view should launch UserActivity

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#32]
```
What steps will reproduce the problem?
1. Open a venue
2. Click checkins tab
3. Click on mayor's section.


This should launch the UserActivity with the USER_ID intent extra.
```

Original issue reported on code.google.com by `jlapenna` on 11 Aug 2009 at 7:00

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 3ac43c5489.
```

Original comment by `jlapenna` on 17 Aug 2009 at 2:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 3ac43c5489.
```

Original comment by `jlapenna` on 5 Jan 2010 at 5:46
